examples/functions/fare/Fare_example_with_park_and_ride.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OJP xmlns="http://www.vdv.de/ojp" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1-dev" xsi:schemaLocation="http://www.vdv.de/ojp ../../../OJP.xsd">
+	<!-- example fare request with a park and ride
+-->
+	<OJPRequest>
+		<siri:ServiceRequest>
+			<siri:RequestTimestamp>2022-01-19T12:00:00Z</siri:RequestTimestamp>
+			<siri:RequestorRef>IRMA</siri:RequestorRef>
+			<OJPFareRequest>
+				<siri:RequestTimestamp>2022-01-19T12:00:00Z</siri:RequestTimestamp>
+				<siri:MessageIdentifier>4712</siri:MessageIdentifier>
+				<TripFareRequest>
+					<Trip>
+						<Id>ID-1232663C-71EF-4E51-8F72-3C0D4A48A809</Id>
+						<Duration>PT38M</Duration>
+						<StartTime>2021-08-20T13:25:00Z</StartTime>
+						<EndTime>2021-08-20T14:03:00Z</EndTime>
+						<Transfers>0</Transfers>
+						<Leg>
+							<Id>1</Id>
+							<ContinuousLeg>
+								<LegStart>
+									<GeoPosition>
+										<siri:Longitude>14.1</siri:Longitude>
+										<siri:Latitude>60.1</siri:Latitude>
+									</GeoPosition>
+									<Name>
+										<Text>Home</Text>
+									</Name>
+								</LegStart>
+								<LegEnd>
+									<PointOfInterestRef>myparkandride</PointOfInterestRef>
+									<Name>
+										<Text>Park and Ride</Text>
+									</Name>
+								</LegEnd>
+								<Service>
+									<PersonalMode>car</PersonalMode>
+								</Service>
+								<Duration>PT15M</Duration>
+							</ContinuousLeg>
+						</Leg>
+						<Leg>
+							<Id>2</Id>
+							<TransferLeg>
+								<TransferType>checkIn</TransferType>
+								<TransferType>parkAndRide</TransferType>
+								<LegStart>
+									<PointOfInterestRef>myparkandride</PointOfInterestRef>
+									<Name>
+										<Text>Park and Ride</Text>
```

Comment:
Funny example - there are no cars allowed on the St. Petersinsel island... :-)

Reply:
I know. What else should I take. I was lazy..... Do we have coordinates and a place in Biel